extensions/github-authentication/src/githubServer.ts

```diff
@@ -243,6 +260,34 @@ export class GitHubServer {
 		} catch (e) {
 			// No-op
 		}
+	}
+
+	public async checkEnterpriseVersion(token: string): Promise<void> {
+		try {
+
+			const result = await fetch(this.getServerUri('/meta').toString(), {
+				headers: {
+					Authorization: `token ${token}`,
+					'User-Agent': 'Visual-Studio-Code'
+				}
+			});
 
+			if (!result.ok) {
+				return;
+			}
+
+			const json: { verifiable_password_authentication: boolean, installed_version: string } = await result.json();
+
+			/* __GDPR__
+				"ghe-session" : {
```

Comment:
Assume this would be valuable information to see where users are coming from. I don't know the schema for GDPR, so adjust as necessary.